benchmarks/sub.mjs

```diff
@@ -817,7 +817,7 @@ const dataSmall = [{
   r: '0x7bffcbe01'
 }];
 
-import JSBI from '../jsbi.mjs';
```

Comment:
It's annoying that TypeScript doesn't support this (I guess?), because it breaks `d8 benchmarks/sub.mjs` — we now have to always test the compiled output instead.